lib/studies/reply/mail_card_preview.dart

```diff
@@ -258,9 +274,15 @@ class _PicturePreview extends StatelessWidget {
 }
 
 class _MailPreviewActionBar extends StatelessWidget {
-  const _MailPreviewActionBar({this.avatar});
+  const _MailPreviewActionBar({
+    @required this.avatar,
+    this.onStar,
+    this.onDelete,
```

Comment:
Will they have to be `() { }` by default so the buttons do not become disabled when `null`?

Reply:
Is setting `(){}` as default necessary, since the only case where the callbacks aren't passed are on mobile, and on mobile the buttons don't appear. The callbacks are always passed on desktop.

Reply:
It is not necessary